code/modules/mob/living/carbon/alien/humanoid/caste/drone.dm

```diff
@@ -4,6 +4,7 @@
 	maxHealth = 100
 	health = 100
 	icon_state = "aliend_s"
+	desc = "A strange alien. This one seems to be less adept at fighting."

```

Comment:
These ones are the builders, they make weeds and nests and walls for the other xenos

Reply:
I did not want to add gameplay details to any of the descriptions. Giant spiders, for example, when examined, only report the color of their eyes. If it's a green one, it doesn't tell you "this spider makes the eggs and has to wrap someone in a web beforehand", it just says "this one has green eyes".

Examining something at a first glance shouldn't inform you of what exactly something is or does.

Reply:
but if you didn't want to add gameplay details then wouldn't "this one seems to be less adept at fighting" be a gameplay detail and not something you see at first glance?

Reply:
You can see at first glance that it doesn't seem built for fighting.

Maybe you know why, maybe you don't. You probably heard thay there are more aliens aboard, and they are very likely stronger than this one.